### PR TITLE
The CI run should also commit the json model dump to gh-pages

### PR DESCRIPTION
**Pull request by maxhbr**
The spec parser can generate a json representation of the content of the md files. This is for example used in https://github.com/spdx/tools-python/pull/726 and it would be helpful if the CI here would generate and persist these files.

### CODE REVIEW

**Comment by zvr** [comment]
The json dump is arbitrary, completely undocumented, and it should not be used for anything. 
The new parser also generates a json dump, but of course the structure is different.

File: .github/workflows/spec-parser.yml [hunk 1]
```diff
@@ -87,13 +88,14 @@ jobs:
...
           cp /tmp/spec-parser.out/context.json .
+          cp /tmp/spec-parser.out/model_dump.json .
 
           rm -rf /auto-generated/
           cp -r /tmp/auto-generated/ .
 
           if [ -n "$(git status -s -- "auto-generated/" "*.ttl" "*.jsonld" "*.json")" ]
           then
-            git add model.ttl model.jsonld context.json auto-generated/
+            git add model.ttl model.jsonld context.json model_dump.json auto-generated/
```
Comment (zvr) [inline .github/workflows/spec-parser.yml:98]: the file is named `model.json` in the new parser

**Review by zvr — CHANGES_REQUESTED** [review]

**Comment by kestewart** [comment]
Per discussion with Joshua in call this is depending on whether the SHACL model is good enough or not.

**Comment by maxhbr** [comment]
this will be replaced by output or use of https://github.com/JPEWdev/shacl2code

**Pull request closed without merging**